Sources/ClientRuntime/Auth/HTTPAuth/DefaultIdentityResolverConfiguration.swift

```diff
@@ -5,14 +5,19 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+import struct Smithy.Attributes
+import struct Smithy.AttributeKey
+import protocol SmithyIdentityAPI.IdentityResolver
+import protocol SmithyIdentityAPI.IdentityResolverConfiguration
+
```

Comment:
Many files contain superficial changes like this, that are related to modularizing the project:
- moved or renamed file
- changed or added imports
- changed namespaces on types
- changed visibility of methods or types (i.e. marking them "public")

I won't comment on these types of changes, since they happen frequently in this PR.